ios/SafeAreaView/RNCSafeAreaContext.mm

```diff
@@ -0,0 +1,71 @@
+#import "RNCSafeAreaContext.h"
+
+#import <SafeAreaContextSpec/SafeAreaContextSpec.h>
+#import <UIKit/UIKit.h>
+
+using namespace facebook::react;
+
+@interface RNCSafeAreaContext () <NativeSafeAreaContextSpec>
+@end
+
+@implementation RNCSafeAreaContext
+
+RCT_EXPORT_MODULE()
+
++ (BOOL)requiresMainQueueSetup
+{
+  return YES;
+}
+
+- (NSDictionary *)constantsToExport
+{
+  return [self getConstants];
+}
+
+- (NSDictionary *)getConstants
+{
+  UIWindow *window = [[UIApplication sharedApplication] keyWindow];
+  if (@available(iOS 11.0, *)) {
```

Comment:
Is iOS 10 still supported?

Reply:
It prob was when the code was written, but it shouldn't be now so I can clean that up.